README.md

```diff
@@ -94,6 +94,24 @@ function CustomTarget() {
 }
 ```
 
+### Use multiple sources
+
+By default only one `Source` is allowed to be injected into a `Target`. Sometimes you may want to inject multiple sources into a single target. Add `multiple` prop on source.
+
+```js
+const Teleporter = createTeleporter()
+
```

Comment:
It should be nice, to allow multiple as an option for createTeleporter: `createTeleporter({ multi: true })`
except if you have use cases in which you could have multiple and non-multiple sources at the same time

Reply:
Yes